README.md

```diff
@@ -394,6 +394,79 @@ This creates a Python entrypoint, YAML configs and a Dockerfile. You can start f
 
 </details>
 
+#### MultimodalDocument
+  
+ A MultimodalDocument in Jina is a document composed by more than one chunks with different modalities.
+ 
+There are different ways to build your multimodal document: 
+ 
+##### From content modality mapping
+Build a Multimodal document by providing the modality names and the content of the chunks.
+  
+```python
+    from jina.types.document.multimodal import MultimodalDocument
+    image_title = 'my holiday picture'
+    image_description = 'the family having fun on the beach'
+    image = PIL.Image.open('path/to/image.jpg')
+    content_mapping = {
+        'title': image_title,
+        'description': image_description,
+        'image': image
+    }   
+    document = MultimodalDocument(modality_content_mapping=content_mapping)
+```
+
+##### From chunks with different modalities
+Build a Multimodal document by creating chunks. Useful when extra metadata needs to exist in every chunk
+  
+```python
+    from jina.types import Document
+    from jina.types.document.multimodal import MultimodalDocument
+    chunk_title = Document('my holiday picture')
+    chunk_title.modality = 'title'
+    
+    chunk_description = Document('the family having fun on the beach')
+    chunk_description.modality = 'description'
+
+    chunk_image = Document(Image.open('path/to/image.jpg'))
+    chunk_image.modality = 'description'
+    chunk_image.tags['date'] = '10/08/2019' 
+  
+    document = MultimodalDocument(chunks=[chunk_title, chunk_description, chunk_image])
+```
+
+#### MultimodalEncoder and MultimodalDriver
+
+In order to extract fusion embeddings from different modalities we have `MultimodalEncoders`.
+An example of a `MultimodalEncoder` can be found in https://github.com/jina-ai/examples/tree/master/multimodal-search-tirg
+
+```yaml
+!TirgMultiModalEncoder
+with:
+  model_path: checkpoint.pth
+  texts_path: texts.pkl
+  positional_modality: ['image', 'text']
+requests:
+  on:
+    [IndexRequest, SearchRequest]:
+      - !MultiModalDriver {}
+```
+
+A `MultimodalEncoder` expects a variable number of arguments in the `encode` interface.
+
```

Comment:
I think it is enough to have this in the documentation and not in the readme.

Reply:
Perhaps we can say "Learn more" and link it to the documentation page.

Reply:
Agreed! I think we need to be very careful about overloading the readme. Better to put a general overview and then a link to more information